### ISSUE TITLE
Сломан на API ≥27

### ISSUE THREAD

**Issue by smbdspk** [saber-nyan/ddlc-android#12]
https://android-developers.googleblog.com/2016/06/android-changes-for-ndk-developers.html
![image 2018-01-09 22 25 27](https://user-images.githubusercontent.com/33251660/34741912-f8bfdbc8-f58c-11e7-84f5-90b2b1d2a5d2.jpg)
[traceback.txt](https://github.com/saber-nyan/ddlc-android/files/1616562/traceback.txt)

Система – AICP Oreo 13.1; Android 8.1; API 27
Железо – Xiaomi mido

**Comment by saber-nyan** [comment]
Точно 23? Сейчас проверю на эмуле.

**Comment by saber-nyan** [comment]
Как минимум API 25 (7.1.1) работает, беру выше...

![API 25](https://user-images.githubusercontent.com/25345196/34742975-2bc08cd2-f599-11e7-9764-70afad7cac5e.png)

**Comment by saber-nyan** [comment]
И API 26 (8.0) тоже работает. Проверяю последний вариант.

**Comment by saber-nyan** [comment]
Да, тост есть. У меня приложение не вылетает, но, скорее всего, будет нестабильно.
Зарепортил здесь: renpy/renpy#1344

**Comment by CPAllianceP** [comment]
хм, всё же только 27. Я предположил по статье в блоге.
А репорты по неофициальным портам не принимают, отклонили уже.

**Comment by saber-nyan** [comment]
> хм, всё же только 27. Я предположил по статье в блоге.
> А репорты по неофициальным портам не принимают, отклонили уже.

Не, это игра пиратская. Я тестовый проект создал и отправил баг, там он тоже есть.
renpy/renpy#1345
https://github.com/saber-nyan/test-renpy-vn

Это ты посты удаляешь, или гитхаб глючит? На емейл все приходит.

**Comment by smbdspk** [comment]
>Это ты посты удаляешь, или гитхаб глючит? На емейл все приходит.

да, я удалил, ибо уже увидел багрепорт по тестовому проекту.

**Comment by saber-nyan** [comment]
Бгг, не удаляй. Пускай история будет.
